### PR TITLE
completed leaderboard front-end feature

### PR DESCRIPTION
**Pull request by VincentWang001**
- Added leaderboard page in "events" section of website
- Added new url "leaderboard" path

### CODE REVIEW

File: hknweb/events/views.py [hunk 1]
```diff
@@ -306,3 +306,22 @@ def send_off_waitlist_email(request, user, event):
...
+        {'name': 'Person', 'points': 18},
+        {'name': 'Zebra', 'points': 100},
+        {'name': 'Penguin', 'points': -18},
+        {'name': 'Elephant', 'points': 20000},
+    ]
+    users.sort(key = lambda user: -user['points'])
+    context = {
+        'users': users
+    }
+    return render(request, 'events/leaderboard.html', context)
```
Comment (ochan1) [inline hknweb/events/views.py:327]: Please include a new line at the end of each file

File: hknweb/events/views.py [hunk 2]
```diff
@@ -306,3 +306,22 @@ def send_off_waitlist_email(request, user, event):
                 settings.NO_REPLY_EMAIL, [user.email])
     msg.attach_alternative(html_content, "text/html")
     msg.send()
+
+
+# Leaderboard
+def show_leaderboard(request):
+    users = [
```
Comment (ochan1) [inline hknweb/events/views.py:313]: TODO: Use real users and have a way to calculate points in the future

(Hint: Consider the "User" Django table)

File: hknweb/events/views.py [hunk 3]
```diff
@@ -306,3 +306,22 @@ def send_off_waitlist_email(request, user, event):
...
+        {'name': 'Abraham', 'points': 10},
+        {'name': 'Beta', 'points': 15},
+        {'name': 'Candy', 'points': 13},
+        {'name': 'Donuts', 'points': 20},
+        {'name': 'Person', 'points': 18},
+        {'name': 'Zebra', 'points': 100},
+        {'name': 'Penguin', 'points': -18},
+        {'name': 'Elephant', 'points': 20000},
+    ]
+    users.sort(key = lambda user: -user['points'])
```
Comment (ochan1) [inline hknweb/events/views.py:323]: Django has a way to sort, so when we filter and extract users, we can use that instead

**Comment by bri25yu** [comment]
Thanks for your work @VincentWang001 :)) Resolved by https://github.com/compserv/hknweb/pull/365

**Pull request closed without merging**